CHANGELOG.md

```diff
@@ -264,6 +264,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed process rank not being available right away after `Trainer` instantiation ([#6941](https://github.com/PyTorchLightning/pytorch-lightning/pull/6941))
 
 
+- Fixed parsing some pre-release version ([#6999](https://github.com/PyTorchLightning/pytorch-lightning/pull/6999))
```

Comment:
This could use a better message

Reply:
any suggestion? :]

Reply:
Fixed parsing for pre-release package versions?